content/en/agent/faq/windows-agent-ddagent-user.md

```diff
@@ -16,42 +17,76 @@ The user `ddagentuser` is created at install time for the Datadog Windows Agent.
+The `<DOMAIN>` can either be a fully-qualified domain name (e.g. `mydomain.com`) or the NETBIOS name (also known as pre-Windows 2000 name).
+It must be separated from the `<USERNAME>` with a `\`.
 
-Additionally, the installer adds the user to the following groups:
+**Note**: The `<USERNAME>` must be 20 characters or less in order to comply with Microsoft's [Active Directory Schema (AD Schema) SAM-Account-Name attribute][1].
 
-* Performance Monitoring
-* Event Log Viewer
+**Note**: Due to a restriction in the MSI installer, the `DDAGENTUSER_PASSWORD` property cannot contain the semicolon character ';'.
 
-**Note**: These options are honored even in a non-domain environment, if the user wishes to supply a username/password to use rather than have the installer generate one.
+#### Domain controllers
 
-**Note**: The `<USERNAME>` must be 20 characters or less in order to comply with Microsoft's [Active Directory Schema (AD Schema) SAM-Account-Name attribute][1].
+##### Primary and backup Domain Controllers
 
-**Note**: When upgrading the Datadog Agent on a domain controller or host where the user has supplied a username for the Agent, you need to supply the `<DDAGENTUSER_NAME>` but not the `<DDAGENTUSER_PASSWORD>`:
+When installing the Agent on a Domain Controller, there is no notion of 'local user', therefore if the installer creates a user it will be a domain user rather than a local one.
 
-**Note**: If encountering permission issues with system and winproc checks upon installing, make sure the `ddagentuser` is a member of the Performance Monitoring and Event Log Viewer groups.
+If no user is specified on the command line, the installer will create a *Domain* account named 'ddagentuser' in the controller's domain.
 
-```shell
-Msiexec /i ddagent.msi DDAGENTUSER_NAME=<DOMAIN>\<USERNAME>
-```
+If a user is specified on the command line, but this user is not found in the Domain, the installer will attempt to create it. If a password was specified, the installer will use that password, otherwise it will generate a random password.
+
+If the specified user is from a parent Domain, the installer will use that user.
+If the user doesn't exist, it will create the user in the child domain (the domain that the controller is joined to).
+The installer will never create a user in the parent Domain.
+
+##### Read-Only Domain Controllers
+
+The installer can only use an existing domain account when installing on a Read-Only Domain Controller.
+
+## Upgrade
+
+When upgrading the Datadog Agent on a domain controller or host where the user has supplied a username for the Agent, it's only required to supply the `DDAGENTUSER_NAME` but not the `DDAGENTUSER_PASSWORD`.
 
 ### Installation with Chef
```

Comment:
This heading level makes it look like this section is a subsection of "Upgrade". Is that accurate, or should this be an H2 `##`?